src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -1681,6 +1681,19 @@ def _openssh_public_key_bytes(self, key):
             )
 
 
+# If the backend has support for Scrypt, dynamically register the backend and
+# implement the interface.
+def _derive_scrypt(self, key_material, salt, length, n, r, p):
```

Comment:
No, because https://github.com/pyca/cryptography/pull/3117/files#diff-2b076058f22fe576413155fbf8ebafbdR19 will catch it if `ScryptBackend` is not registered so even though the method is present it won't be called.